raw_description/urdf/base_long/base.urdf.xacro

```diff
@@ -117,12 +117,12 @@
 
 
     <!-- base laser front -->
-    <xacro:sick_s300_laser_v0 name="${name}_laser_front" parent="${name}" ros_topic="scan_front_raw" update_rate="10" min_angle="-2.3562" max_angle="2.3562" >
+    <xacro:sick_s300_laser_v0 name="${name}_laser_front" parent="${name}" ros_topic="laser_front/scan_raw" update_rate="10" min_angle="-2.3562" max_angle="2.3562" >
```

Comment:
Since no `laserName` or something like that is set in [sick_s300_laser](https://github.com/ipa-srd-fg/cob_common/blob/indigo_dev/cob_description/urdf/sensors/sick_s300_laser.gazebo.xacro), there is also no concatenation.